warehouse/templates/packaging/detail.html

```diff
@@ -400,6 +409,82 @@ <h3>
               {% endtrans %}
             </h3>
 
+              <p>
+                Filter files by name,
+                <abbr title="Interpreter implementation and version">interpreter</abbr>,
+                <abbr title="Application binary interface">ABI</abbr>,
+                and platform.
+              </p>
+
+              <p>
+                Showing
+                <span data-filter-list-target="shown"></span>
+                of
+                <span data-filter-list-target="total"></span>
+                files.
+              </p>
```

Comment:
This is going to be fairly tricky to translate because it a) includes embedded HTML and b) needs to handle pluralizations c) the pluralized value is being determined on the frontend after the HTML has already been rendered, so Jinja's `{% pluralize %}` tag won't work here.

I think we might want to define these strings into the JS source instead, which will allow us to use `ngettext` on them there, but we don't currently do that in this codebase I believe, so it will probably require figuring out how to start doing that.